assets/scss/modal.scss

```diff
@@ -26,4 +26,67 @@
     justify-content: space-between;
     padding: 1rem 0;
   }
+
+  &#shareModal {
+    .modal-dialog {
+      max-width: 560px;
+      margin-top: 10rem;
+    }
+    .modal-content {
+      border: none;
+      .modal-header {
+        border-bottom: none;
+        padding: 1.5rem 1.5rem 0 1.5rem;
+      }
+      .modal-title {
+        font-size: 2.125rem;
+        font-weight: 600;
+      }
+      .modal-body {
+        padding: 1rem 1.5rem 1.5rem 1.5rem;
+        .icon-wrapper > div {
+          display: flex;
+          justify-content: space-between;
+        }
+        a {
+          border-radius: 0.25rem;
+          width: calc(100% / 3 - 10px);
+          height: 3rem;
+          &.facebook {
```

Comment:
Do these need to be here in the shared SCSS or should they be scoped within the component?

Reply:
they can be moved to the component, however, adding the scoped attribute to components scss means this style is not applied to the component. I have moved the style to the socialShareModal component but without the scoped attribute